content/_index.md

```diff
@@ -41,80 +41,19 @@ sections:
-                    <img class="logo col-6 col-md-4" src="/images/logos/project/binder.svg" alt="Binder logo">
-                </div>
-            </div>
-        </div>
-
-
-  - block: markdown
-    id: numbers
-    content:
-      title: We serve a global network of communities
-      subtitle: 
-      text: |
-
-        We serve **over 80 communities across the globe** with **over 7000 active users** that are dedicated to creating and sharing knowledge. This includes **educational communities** that share knowledge within a community, **research communities** that create advances at the frontier of knowledge, and **applied knowledge communities** that use advances in knowledge to serve the public.
-
-        <style>
-          #who-logos {
-            display: flex;
-            flex-wrap: wrap;
-            align-items: center;
-            gap: 1em;
-          }
-          .list-inline-item {
-            flex: 1
-          }
-
-          .list-inline-item img {
-            filter: grayscale(100%); /* Desaturates the image completely */
-          }
-        </style>
-
-        {{< figure src="/images/home/worldmap.png" alt="2i2c map of communities" caption="A map of the rough location of communities with currently active hubs. Updated as of April 2024, [see our interactive map](https://2i2c.org/kpis/cloud/#map-of-hubs) for the latest data.">}}
-
-        </p>
-
-        <ul id="who-logos" class="list-inline">
-            <li class="list-inline-item"><a href="https://nasa.gov" target="_blank"><img src="/images/logos/community/nasa.svg" alt="Nasa logo"></a></li>
-            <li class="list-inline-item"><a href="https://www.si.edu/" target="_blank"><img src="/images/logos/community/smithsonian.svg" alt="Smithsonian logo"></a></li>
-            <li class="list-inline-item"><a href="https://www.hhmi.org/" target="_blank"><img src="/images/logos/community/howard-hughes-medical-institute.svg" alt="Howard Hughes Medical Institute logo"></a></li>
-            <li class="list-inline-item"><a href="https://ncas.ac.uk/" target="_blank"><img src="/images/logos/community/ncas.png" alt="NCAS logo"></a></li>
-            <li class="list-inline-item"><a href="https://pangeo.io/" target="_blank"><img src="/images/logos/community/pangeo.svg" alt="Pangeo logo"></a></li>
-            <li class="list-inline-item"><a href="https://www.utoronto.ca/" target="_blank"><img src="/images/logos/community/university-of-toronto.svg" alt="University of toronto logo"></a></li>
-            <li class="list-inline-item"><a href="https://columbia.edu/" target="_blank"><img src="/images/logos/community/columbia-university.png" alt="Columbia University logo"></a></li>
-        </ul>
+        Our platform **empowers community leaders in research and education to design a hub** for their community to create and share knowledge. We do so by bringing together open tools, services, data, and infrastructure.
 
+        {{< servicetech >}}
```

Comment:
This renders differently on Safari and Firefox (not checked Chrome).

<img width="1392" alt="Screenshot 2024-07-06 at 10 27 16" src="https://github.com/2i2c-org/2i2c-org.github.io/assets/45105935/803442fb-6066-467b-b291-c5a8bf436ef6">
<img width="1391" alt="Screenshot 2024-07-06 at 10 27 44" src="https://github.com/2i2c-org/2i2c-org.github.io/assets/45105935/241c5150-636c-4800-94ed-aa8da929b5a6">